### PR TITLE
Library files from https://github.com/jrosain/Homotopy-Finiteness/

### PR DESCRIPTION
**Pull request by jrosain**
Organized the files from my internship in a coherent file system & split up the prelude to integrate it seamlessly.

Added a file `Everything.ctt` for testing purposes. It should typecheck 1334 definitions.

### CODE REVIEW

File: library/Foundation/.#Paths.ctt
Comment (JonasHoefer) [inline library/Foundation/.#Paths.ctt:1]: I think this file is not supposed to be part of the PR.

**Review by JonasHoefer — CHANGES_REQUESTED** [review]
Except for the minor comment below, the PR looks very good!

File: library/Foundation/.#Paths.ctt
Comment (jrosain) [reply library/Foundation/.#Paths.ctt:1]: That's right, it has been removed in 36ef90f.

**Review by JonasHoefer — APPROVED** [review]
Looks good now. Thank you very much for all your hard work on the project!

**Pull request merged**